test/sample-test.ts

```diff
@@ -16,46 +51,54 @@ describe("Starknet", function () {
     const contract: StarknetContract = await contractFactory.deploy();
```

Comment:
This `it` block says that it `should work for a fresh deployment`. It's not informative of the actual purpose of this test and should be changed.